server/storage_config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "location": "./file_uploads",
+  "delete_on_edit": true
```

Comment:
What does `delete_on_edit` do?

Reply:
When you edit and upload a new file, `delete_on_edit` determines whether to delete the old file or not.